### ISSUE TITLE
Add pin layout screens for PSLab v6

### ISSUE THREAD

**Issue by marcnause** [fossasia/pslab-app#2499]
**Actual Behaviour**

The app contains pin layout screens for PSlab v5.

**Expected Behaviour**

The app contains pin layout screens for PSlab v5 ans PSlab v6.

**Steps to reproduce it**

1. Start app
2. Press menu symbol (3 dots) in upper right
3. Menu entries lead to pin layout screens for PSlab v5 only.

**Would you like to work on the issue?**

Maybe.

**Comment by Surajkumar5050** [comment]
I would like to work on this issue. Please assign it to me, and I'll start by analyzing the current implementation for PSLab v5 and adding support for PSLab v6 pin layout screens.

**Comment by AsCress** [comment]
@Surajkumar5050 Sure! You can go ahead and work on this.

**Comment by samruddhi-Rahegaonkar** [comment]
@Surajkumar5050 Are you working on this issue ? 
@AsCress If he is not working I would like to solve this issue.

**Comment by Surajkumar5050** [comment]
Hi @samruddhi-Rahegaonkar , my exams are currently going on, so please feel free to work on this issue. You can ask @AsCress for this.

**Comment by samruddhi-Rahegaonkar** [comment]
@AsCress Should I work on this issue?

**Comment by AsCress** [comment]
@samruddhi-Rahegaonkar Sure! Go ahead.

**Comment by marcnause** [comment]
Just in case you have not seen it yet, images of the PSLab v6 board can be found at https://github.com/fossasia/pslab-hardware

**Comment by samruddhi-Rahegaonkar** [comment]
Thanks! @marcnause Was looking for it only.

**Comment by samruddhi-Rahegaonkar** [comment]
@AsCress Started working on this issue. any Guidelines for this?

**Comment by samruddhi-Rahegaonkar** [comment]
@AsCress which editor has been used for color map for v5 image ?

**Comment by AsCress** [comment]
@samruddhi-Rahegaonkar How are you approaching this ?
I may not have an idea about the editor that was used for the color map. 
@CloudyPadmal Do you have an idea about this ?

**Comment by samruddhi-Rahegaonkar** [comment]
@AsCress All the process will same as v5 just need to create new xml and java files for v6 and creating new color map for v6 and pin informations dialog box after clicking on each pin.

**Comment by AsCress** [comment]
@samruddhi-Rahegaonkar That's great. And how would the user be switching between the pin layouts for the _v6_ and the _v5_ ?

**Comment by samruddhi-Rahegaonkar** [comment]
@AsCress User can switch through `pslab_connectivity_menu.xml` we can add two more menus below v5 for v6.

**Comment by AsCress** [comment]
> @AsCress User can switch through `pslab_connectivity_menu.xml` we can add two more menus below v5 for v6.

Yup. This would be nice to start with. Go ahead and implement whatever you find best.

**Comment by CloudyPadmal** [comment]
> @samruddhi-Rahegaonkar How are you approaching this ?
> 
> I may not have an idea about the editor that was used for the color map. 
> 
> @CloudyPadmal Do you have an idea about this ?
> 
> 

I used Inkscape for the design

**Comment by samruddhi-Rahegaonkar** [comment]
Hello @AsCress ,
I have to use the same colours in the color map which is given in the following pdf ?

<img width="1280" alt="Image" src="https://github.com/user-attachments/assets/3760641a-885a-4bcd-bdb4-eea255fc2345" />

**Comment by AsCress** [comment]
@samruddhi-Rahegaonkar Yes. This looks nice !

**Comment by samruddhi-Rahegaonkar** [comment]
@marcnause @AsCress I am done with the work. should I send PR ?

**Comment by AsCress** [comment]
> [@marcnause](https://github.com/marcnause) [@AsCress](https://github.com/AsCress) I am done with the work. should I send PR ?

Go for it !

**Comment by rahul31124** [comment]
Closing this issue as not relevant as anymore
